frontend/src/app/actions/newsletter-form.js

```diff
@@ -1,4 +1,7 @@
 import { createActions } from 'redux-actions';
+import 'isomorphic-fetch';
```

Comment:
Even though this call defines a global in the bundled module, it's necessary for this to be imported in the module for the mock to work. It shouldn't affect the bundle.